### PR TITLE
Added French (fr) translations

### PR DESCRIPTION
**Pull request by sagzy**
no issue
- Used informal form ("tu")

### CODE REVIEW

File: ghost/i18n/lib/i18n.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const i18next = require('i18next');
 
-const SUPPORTED_LOCALES = ['en', 'nl'];
+const SUPPORTED_LOCALES = ['en', 'nl', 'fr'];
```
Comment (naz) [inline ghost/i18n/lib/i18n.js:3]: let's make it a multi-line array declaration straight away. I imagine every single translation will be modifying this one line causing lots of conflicts.

Comment (daniellockyer) [reply ghost/i18n/lib/i18n.js:3]: ➕ I would agree, nice idea

Comment (daniellockyer) [reply ghost/i18n/lib/i18n.js:3]: + alphabetically sorted? 😬

Comment (sagzy) [reply ghost/i18n/lib/i18n.js]: yes, good idea! Updated :)

**Pull request merged**